Packs/Cybereason/Integrations/Cybereason/Cybereason_test.py

```diff
@@ -260,3 +260,49 @@ def test_query_file(mocker):
     assert 'Cybereason file query results' in result[0]['HumanReadable']
     assert result[0]['EntryContext']['Cybereason.File(val.MD5 && val.MD5===obj.MD5 || val.SHA1 && '
                                      'val.SHA1===obj.SHA1)'][0]['Machine'] == 'desktop-p0m5vad'
+
+
+def test_login_failed(requests_mock, mocker):
+    """
+    Given:
+        - Cybereason instance with invalid credentials
+
+    When:
+        - Running test module
+
+    Then:
+        - Ensure an indicative error is returned that authorization failed
+    """
+    login_failed_html = """<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>Cybereason | Login</title>
+    <base href="/">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="icon" href="favicon.ico">
+<link rel="shortcut icon" href="favicon.ico"><link href="public/vendors_c29907a62751511cc002.css" rel="stylesheet"><link href="public/login_62faa8ec0f21f2d2949f.css" rel="stylesheet"></head>  # noqa: E501
+<body class="cbr-theme-dark">
+    <app-login></app-login>
+<script type="text/javascript" src="public/vendors_c29907a62751511cc002.js"></script><script type="text/javascript" src="public/login_62faa8ec0f21f2d2949f.js"></script></body>  # noqa: E501
+</html>
+""".encode('utf-8')
+    mocker.patch.object(demisto, 'params', return_value={
+        'server': 'http://server',
+        'credentials': {
+            'identifier': 'username',
+            'password': 'password'
+        },
+        'proxy': True
+    })
+    mocker.patch.object(demisto, 'command', return_value='test-module')
+    return_error_mock = mocker.patch('Cybereason.return_error')
+    requests_mock.post('http://server/login.html', content=login_failed_html)
+    requests_mock.post('http://server/rest/visualsearch/query/simple', content=login_failed_html)
```

Comment:
Why do we need this mock? after a failure in login() shouldn't an exception be raised?

Reply:
@ShahafBenYakir it is a bit misleading - the login will never actually fail, as it always return an HTML (and hence we can't really verify it), unlike the actual API requested which return an HTML only in case of failure